jsapp/js/actions.es6

```diff
@@ -383,17 +383,21 @@ actions.resources.listTags.completed.listen(function(results){
   }
 });
 
-actions.resources.updateAsset.listen(function(uid, values, params={}) {
+actions.resources.updateAsset.listen(function(uid, values, callbacks={}) {
```

Comment:
We shouldn't change `params` to `callbacks` here, and we shouldn't change `onComplete` to `onCompleted` below. It's not necessary to make these changes to nomenclature, it is longer to review and it's also prone to merge conflicts (the `searchStore` branch makes changes in the same lines, roughly).